src/containers/chat/chat-item/ChatItem.tsx

```diff
@@ -0,0 +1,85 @@
+import { FC, memo } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Box from '@mui/material/Box'
+import Avatar from '@mui/material/Avatar'
+import Typography from '@mui/material/Typography'
+
+import { useAppSelector } from '~/hooks/use-redux'
+
+import { styles } from '~/containers/chat/chat-item/ChatItem.styles'
+import { ComponentEnum, LatestMessage, UserResponse } from '~/types'
+import { getFormatedDate, spliceSx } from '~/utils/helper-functions'
+
+interface ItemOfChatProps {
+  user: Pick<UserResponse, '_id' | 'firstName' | 'lastName' | 'photo'>
+  lastMessage: LatestMessage
+  isActiveChat: boolean
+  setIsSelectedChat: (id: string) => void
+  closeDrawer?: () => void
+}
+
+const ChatItem: FC<ItemOfChatProps> = ({
+  user,
+  lastMessage,
+  isActiveChat,
+  setIsSelectedChat,
+  closeDrawer
+}) => {
+  const { t } = useTranslation()
+  const { userId } = useAppSelector((state) => state.appMain)
+
+  const { firstName, lastName, photo } = user
+  const { text, chat, author, updatedAt } = lastMessage
+
+  const fullName = `${firstName} ${lastName}`
+
+  const handleSelectedChat = () => {
+    setIsSelectedChat(chat)
+    closeDrawer && closeDrawer()
+  }
+
+  const formattedTime = getFormatedDate({
+    date: updatedAt,
+    locales: 'en-GB',
+    options: {
+      day: '2-digit',
+      month: '2-digit',
+      year: 'numeric'
+    },
+    isCurrentDayHours: true
+  })
+
+  const activeChat = isActiveChat ? styles.activeChat : undefined
```

Comment:
As I see, you have only backgroundColor property in this style, maybe better to move this property to root styles and pass isActiveChat property:  

styles.root(isActiveChat)

Reply:
Fixed